_docs-2/administration/fate.md

```diff
@@ -71,9 +71,15 @@ Sometimes, it is useful to inspect the current FATE operations, both pending and
 For example, a command that is not completing could be blocked on the execution of another
 operation. Accumulo provides an Accumulo shell command to interact with fate.
 
-The `fate` shell command accepts a number of arguments for different functionality:
-`list`/`print`, `summary`, `cancel`, `fail`, `delete`, `dump`.
+The `fate` admin command accepts a number of arguments for different functionality:
+`list`/`print`, `summary`, `cancel`, `fail`, `delete`, `dump`.  
 
+The command for launching the fate admin command is:
```

Comment:
I think some of this reads better if we put some blank lines between markdown paragraphs and code blocks... and before the next header.
```suggestion
The command for launching the fate admin command is:

```